test/models/dish_menu_test.rb

```diff
@@ -18,15 +18,15 @@ class DishMenuTest < ActiveSupport::TestCase
     test 'get menu' do
         Dish.add_dish(name: "french toast", star: 3, mesh_soft:2, puree:"", portion_size:"", diet:"1/2 serving")
         DishMenu.add_dish_to_cycle(48,"french toast")
-        menu=DishMenu.get_menu(48)
+        menu=DishMenu.get_by_day(48)
         assert menu.length()==1
```

Comment:
Clarification question: how do we know that the menu will be empty on day 48 before you add the dish?